README.md

```diff
@@ -1,24 +1,190 @@
+Refugee Aid is an app that connects citizens in the United States with refugee aid organizations and camps.
 
-Things you may want to cover:
+## :clipboard: Table of Contents
+[:hammer: Installation](#hammer-installation) | [:question: GraphQL Queries](#question-graphql-queries) | [:people_holding_hands: Authors](#people_holding_hands-authors)
 
-* Ruby version
+## :hammer: Installation
 
-* System dependencies
+Install all necessary gem files:
+```ruby
+bundle install
+```
 
-* Configuration
+Create, migrate, and seed the PostgresQL database:
+```ruby
+bundle exec rails db:{create,migrate,seed}
+```
 
-* Database creation
+Run the test suite to verify that everything works:
+```ruby
+bundle exec rspec
+```
 
-* Database initialization
+---
 
-* How to run the test suite
+## :question: GraphQL Queries
 
-* Services (job queues, cache servers, search engines, etc.)
+| Types | Description |
+| --- | ---|
+| [`Organization`](#organization) | The entity requesting aid. |
+| [`AidRequest`](#aidrequest) | The aid an organization is requesting. |
 
-* Deployment instructions
+#### Organization
 
-* ...
+```graphql
+type Organization {
+  id: ID!
+  name: String
+  city: String
+  contactEmail: String
+  contactPhone: String
+  website: String
+  streetAddress: String
+  city: String
+  state: String
+  zip: String
+  latitude: Float
+  longitude: Float
+  shareAddress: Boolean
+  shareEmail: Boolean
+  sharePhone: Boolean
+  aidRequests: [AidRequest]
+}
+```
+
+#### AidRequest
+
+```graphql
+type AidRequest {
+  id: ID!
+  organizationId: Integer
+  aidType: String
+  language: String
+  description: String
+  status: String
+  organization: Organization!
+}
+```
+
+
+<table>
+<tr>
+<th width="500px">
+<small>Query</small>
+</th>
+<th width="500px">
+<small>
+Response
+</small>
+</th>
+</tr>
+<tr>
+<td>
+  
+```graphql
+{
+  organization(id: 1) {
+    id                # The organization's ID
+    name              # The name of the organization
+    contactPhone      # A contact phone number
+    contactEmail
+    streetAddress
+    website
+    city
+    state
+    zip
+    latitude
+    longitude
+    shareAddress
+    sharePhone
+    shareEmail
+    aidRequests {
+      id
+      organizationId
+      aidType
+      language
+      description
+      status
+    }
+  }
+}
+```
+
+</td>
+<td>
+
+```json
+{
+  "data": {
+    "organization": {
+      "id": "1",
+      "name": "Welcome Winds Refuge",
+      "contactPhone": "1-347-839-2631",
+      "contactEmail": "mitchel.schmitt@powlowski.test",
+      "streetAddress": "8223 Lou Crest",
+      "website": "http://kemmer.test/carita",
+      "city": "Oklahoma City",
+      "state": "OK",
+      "zip": "49131",
+      "latitude": 35.653564198354196,
+      "longitude": -97.19134672640317,
+      "shareAddress": false,
+      "sharePhone": true,
+      "shareEmail": true,
+      "aidRequests": [
+        {
+          "id": "2000",
+          "organizationId": 1,
+          "aidType": null,
```

Comment:
This should be fixed in the db now after we merge #28! So it won't return `nil` 
I guess you can just update these with a random language